2007-11-12  Aurelien Jarno  <aurelien@aurel32.net>
	    Thomas Schwinge  <tschwinge@gnu.org>

	* sysdeps/mach/i386/sysdep.h (PTR_MANGLE, PTR_DEMANGLE): Define.

---
 sysdeps/mach/i386/sysdep.h |    9 +++++++++
 1 file changed, 9 insertions(+)

--- a/sysdeps/mach/i386/sysdep.h
+++ b/sysdeps/mach/i386/sysdep.h
@@ -55,6 +55,15 @@
 
 #define STACK_GROWTH_DOWN
 
+/* Pointer mangling is not yet supported for Hurd.  */
+#ifdef __ASSEMBLER__
+#define PTR_MANGLE(var)
+#define PTR_DEMANGLE(var)
+#else
+#define PTR_MANGLE(var) (void) (var)
+#define PTR_DEMANGLE(var) (void) (var)
+#endif
+
 /* Get the machine-independent Mach definitions.  */
 #include <sysdeps/mach/sysdep.h>
 
